src/Network/Network/Models/PSAvailableAlias.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSAvailableAlias
```

Comment:
Is there a reason why it's `AvailableAlias` instead of `AvailableServiceAlias`?

Reply:
> Is there a reason why it's AvailableAlias instead of AvailableServiceAlias?

Changed.